src/components/victory-area/helper-methods.js

```diff
@@ -6,6 +6,13 @@ import Scale from "../../helpers/scale";
 
 export default {
 
+  getScale(props, fallbackProps) {
+    const modifiedProps = Helpers.modifyProps(props, fallbackProps);
+    const {scale} = this.getCalculatedValues(modifiedProps);
```

Comment:
I would prefer if getScale() were broken out into its own method and then used in `getCalculatedValues` instead of the other way around so we can avoid unnecessary calculations